src/main/flight/pid_mw23.c

```diff
@@ -146,10 +149,14 @@ void pidMultiWii23(const pidProfile_t *pidProfile, const controlRateConfig_t *co
         // Delta from measurement
         delta = -(gyroError - lastErrorForDelta[axis]);
         lastErrorForDelta[axis] = gyroError;
-        if (pidProfile->dterm_cut_hz) {
-            // Dterm delta low pass
-            DTerm = delta;
-            DTerm = lrintf(applyBiQuadFilter((float) DTerm, &deltaFilterState[axis])) * 3;  // Keep same scaling as unfiltered DTerm
+        if (pidProfile->dterm_lpf_hz) {
+            // Dterm low pass filter
+            DTerm = delta * 3; // Keep same scaling as unfiltered DTerm
+#ifdef USE_PID_BIQUAD_FILTER
```

Comment:
2nd order filter (biquad) with double bandwidth may be better then first order one and delay will be similar ...